plugins/modules/packaging/language/pear.py

```diff
@@ -31,11 +31,24 @@
         default: "present"
         choices: ["present", "absent", "latest"]
     executable:
-      description:
-        - Path to the pear executable
+        description:
+            - Path to the pear executable
+    prompts:
+        description:
+            - List of regular expressions that can be used to detect prompts during pear package installation to answer the expected question.
+            - Prompts will be processed in the same order as the packages list
```

Comment:
```suggestion
            - Prompts will be processed in the same order as the packages list.
```
(Same for next ones.)